core/tools/messagehandler/messagehandler.cpp

```diff
@@ -149,10 +149,10 @@ MessageHandler::MessageHandler(Probe *probe, QObject *parent)
     // install handler directly, catches most cases,
     // i.e. user has no special handler or the handler
     // is created before the QApplication
-    ensureHandlerInstalled();
+//     ensureHandlerInstalled();
     // recheck when eventloop is entered, if the user
     // installs a handler after QApp but before .exec()
-    QMetaObject::invokeMethod(this, "ensureHandlerInstalled", Qt::QueuedConnection);
+//     QMetaObject::invokeMethod(this, "ensureHandlerInstalled", Qt::QueuedConnection);
```

Comment:
Are these changes intentional or should this be conditional on the Qt version?

Reply:
done.